pkg/api/handlers/projects.go

```diff
@@ -358,6 +358,10 @@ func (h *ProjectsHandler) CreateProject(req api.Context) error {
 			DefaultModel:         project.DefaultModel,
 			Models:               project.Models,
 		},
+		// Set the last used time to now, so that the project will get cleaned up by retention, even if it is never used.
+		Status: v1.ThreadStatus{
+			LastUsedTime: metav1.Now(),
+		},
```

Comment:
Seems like this is already handled here: https://github.com/obot-platform/obot/blob/main/pkg/controller/handlers/retention/retention.go#L74-L77

I am seeing that my newly created, not chatted with threads, have this field set.

Reply:
Hmm I didn't think about this. Let me try something...

Reply:
You're right, this PR doesn't do anything. Closing.